### ISSUE TITLE
-q quitOnError option is now --warningAsErrors user guide is outdated

### ISSUE THREAD

**Issue by blastmaster** [hpc/ior#256]
Hi,
the `-q` `quitOnError` option is no longer supported and leads to an unrecognized option error.
I'am fine with the change also with `ior` behavior, but it is still noted in  the [`user doc`](https://github.com/hpc/ior/blob/master/doc/USER_GUIDE#L95). 
Maybe you might want to update the user guide.

**Comment by JulianKunkel** [comment]
Agreed, the user doc should be updated. Note that these options do something slightly different.
The -q option only stopped during buffercomparison (any error during a read or write check), while the --warningAsErrors option stops on an IOR warning (which includes read or write check). The reason is that warnings are often involved in performance changing behavior.

**Comment by jschwartz-cray** [comment]
> Note that these options do something slightly different. The -q option only stopped during buffercomparison (any error during a read or write check)

So what is the default behavior now?  Like -q was specified, or not?

> the --warningAsErrors option stops on an IOR warning (which includes read or write check)

If this implies the default is now as though -q was not specified, what if I want to stop on read/write check errors, but not on "performance changing behavior"?
